src/instructlab/eval/exceptions.py

```diff
@@ -22,6 +22,21 @@ def __init__(self, path) -> None:
         self.message = f"Model could not be found at {path}"
 
 
+class JSONNotFoundError(EvalError):
+    """
+    Error raised when JSON in path is not able to be found
+
+    Attributes
+        message     error message to be printed on raise
+        path        filepath of model location
+    """
+
+    def __init__(self, path) -> None:
+        super().__init__()
+        self.path = path
+        self.message = f"JSON could not be found at {path}"
```

Comment:
I am not sure what I would do is I was a user and saw this error given I didn't pass in a json file.  Would an exception like "InvalidModelError" and a message like:

The content found at {path} does not appear to be a supported model type

Reply:
This I think is directly related to finding a `config.json` in a safetensors directory - maybe something related to that?

Reply:
Will push up something like this for further review!